results/debug-oracle.json

```diff
@@ -153,7 +153,7 @@
           "generation_metadata": {
             "score": -0.48298680782318115,
             "dropped_seqs": [],
-            "n_words_checked": 77
+            "n_words_checked": 65
```

Comment:
This has been reduced due to the bugfix in this commit: https://github.com/danton-nlp/factual-beam-search/pull/50/commits/f238b1b66f46312738c69c4bbe70d098ff0621ec